### DISCUSSION TITLE
Teampass 3.0.0.22

### DISCUSSION THREAD

**Discussion by nilsteampassnet** [nilsteampassnet/TeamPass#3519] [General]
# Please ALWAYS download from [the latest commit](https://github.com/nilsteampassnet/TeamPass/archive/refs/heads/master.zip) as this project is evolving regularly.

## What's new
This release embeds next fixes:

- Requires GMP extension for PHP
- All language files updated with most recent translations from POEditor
- Improving tasks feature with real background execution. It is highly recommended to enable it for better performance.
- Docker file updated
- Improved folders management page
- Improved cache system
- Lot of bug fix
- API - item details
- Full list of changes: [3.0.0.21 -> 3.0.0.22](https://github.com/nilsteampassnet/TeamPass/compare/3.0.0.21...3.0.0.22?expand=1)

## Limitation

All language strings are not yet updated.
Please join [Teampass v3 translation project on Poeditor](https://poeditor.com/projects/view?id=433631) and participate in its translation in your language.

## Upgrading from 2.1.27.36

The process might take a long time. Indeed, the encryption process has totally changed for improving the security.
Please have a look at [the new encryption process schematic](https://preview.redd.it/5x3w5fhis4l21.png?width=1051&format=png&auto=webp&s=4c2f08ef9f4b928b9eb4b2e68f985f72c4a7d037).
As a consequence, the upgrade process will regenerate all the expected keys for all items and users, which might take a long time.

Any feedback is very welcome.

<hr /><em>This discussion was created from the release <a href='https://github.com/nilsteampassnet/TeamPass/releases/tag/3.0.0.22'>Teampass 3.0.0.22</a>.</em>

**Comment by odanielbarbieri** [comment]
Hello! I'm having a hard time. When I create a new item, after creating this new item, the system asks me to inform the new password, but I didn't change the password...

![image](https://user-images.githubusercontent.com/116836106/218789597-5216237a-1f8c-4ddc-b4bb-13f7d0d9809a.png)


Can you help me? The version is 3.0.0.22

**Reply by nilsteampassnet** [reply]
Please rise a bug for specific follow-up.

**Reply by Kyogre** [reply]
hi! I have rised it recently - https://github.com/nilsteampassnet/TeamPass/issues/3604
